src/main/java/nextstep/courses/domain/SessionStudent.java

```diff
@@ -0,0 +1,35 @@
+package nextstep.courses.domain;
+
+import nextstep.users.domain.NsUser;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class SessionStudent {
+
+    private final int availableSeats;
+    private List<NsUser> students = new ArrayList<NsUser>();
```

Comment:
Nit: Java 1.7부터는 컴파일러가 앞서 선언된 타입을 통해 타입 추론이 가능하기 때문에 뒤에서는 생략이 가능합니다.
```suggestion
    private List<NsUser> students = new ArrayList<>();
```